crates/extensions/c8y_http_proxy/src/actor.rs

```diff
@@ -393,8 +394,7 @@ impl C8YHttpProxyActor {
                 end_point.get_url_for_event_binary_upload(&event_response_id);
             Ok(HttpRequestBuilder::post(&binary_upload_event_url)
                 .header("Accept", "application/json")
-                .header("Content-Type", "text/plain")
```

Comment:
This should be added back in otherwise c8y responds with a 415 status code, though use a Content-Type of `application/octet-stream` instead.

Reply:
Fixed by 6d7a38f854b8c99c125503194fa1fd9ee82f710f